guide/payments/transactions.md

```diff
@@ -121,3 +121,5 @@ The sender's wallet uses both of these as inputs in the transaction, meaning tha
 Every transaction needs to pay a fee to incentivize miners to include it in a block. There is no fixed fee for making a transaction as it depends on the amount of data it includes, the amount of other transactions that are trying to get verified, and how much each submitter is prepared to pay. Miners typically pick the transactions that will earn them the highest reward to include in a block.
 
 Blocks are also limited in size and new ones are created every 10 minutes on average. This means that if you want a transaction to be confirmed in the next block you might have to pay a relatively high price.
+
```

Comment:
```suggestion

---

```

A horizontal rule is typically used to separate this statement from the above content. e.g. https://bitcoin.design/guide/getting-started/visual-language/

Reply:
It's used on all the other page to page links

Reply:
The text is fine, just need the line spacer. Could you add it in? Should look like this.

<img width="907" alt="Screenshot 2021-05-10 at 13 17 51" src="https://user-images.githubusercontent.com/183140/117651365-29d37e80-b192-11eb-9c5c-4d9117234fb6.png">